node-admin/src/test/java/com/yahoo/vespa/hosted/node/admin/integrationTests/StorageMaintainerMock.java

```diff
@@ -45,4 +46,8 @@ public void cleanNodeAdmin() {
     public void cleanupNodeStorage(ContainerName containerName, ContainerNodeSpec nodeSpec) {
         callOrderVerifier.add("DeleteContainerStorage with " + containerName);
     }
+
+    @Override
+    protected void writeSecretAgentConfig(ContainerName containerName) throws IOException {
```

Comment:
stray override?

Reply:
Yes. Tried to map the file in and then write it using `Files.write()` and created Override so the tests don't break. Looks like the tests work fine without it when writing using `dockerOperations`.